### PR TITLE
feat : added Marissa.md

### PR DESCRIPTION
**Pull request by marissahuysentruyt**
## Description
-Create new feature branch: feat--2022-marissa
-Add Marissa.md file
-Completed aboutme questions

#### To Validate
  1. Make sure all GitHub action PR checks have passed.
  2. Pull down all related branches.
  3. Feel free to spell check

### CODE REVIEW

File: about-the-apprentices/2022/Marissa.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Hello there
+
+Hello everyone! My name is Marissa and I'm a new apprentice at Sparkbox! Right now, I'm working remotely from Troy, MI, which means I'm on Eastern time and my preferred pronouns are she/her/hers.
+
+One of my favorite things about programming is the building and creating aspect. One thing I'm excited about learning is React...and Git...and anything backend related since I don't have any experience with that...there's a lot I'm excited about!.
```
Comment (jkohrt7) [inline about-the-apprentices/2022/Marissa.md]: You've got an extra period at the end of this line!

File: about-the-apprentices/2022/Marissa.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## Icebreakers
+
+* What technology innovation made the most impact on your life
+I don't have a great answer for that yet. I'm part of that generation that sort of grew up analog, and transitioned into digital. The access to an abundance of information on the web is something that we just take for granted now, but it wasn't really like that even 30 years ago! 
+
+* Desktop, laptop, or tablet? Which do you prefer to use?
+Laptop, but with an extra monitor :fire:
```
Comment (jkohrt7) [inline about-the-apprentices/2022/Marissa.md:13]: More monitors > fewer monitors! 👍

**Review by jkohrt7 — CHANGES_REQUESTED** [review]
After you remove that period on line 5, I'll go ahead and merge the branches.

File: about-the-apprentices/2022/Marissa.md (hunk 1 above)
Comment (marissahuysentruyt) [reply about-the-apprentices/2022/Marissa.md]: good catch! should be fixed i hope!

**Review by jkohrt7 — APPROVED** [review]
Punctuation's been fixed, looks fine now 👍

**Comment by jordanleven** [comment]
This looks great! @marissahuysentruyt   - feel free to squash this into a single command and push up. After that, @jkohrt7  can pull down the squashed branch by using `git pull --rebase` and then merge it into `main` by running `git merge feat--2022-marissa --ff-only`.

**Pull request merged**